astropy/units/tests/test_quantity_ufuncs.py

```diff
@@ -88,6 +92,30 @@ def test_removal_addition(self):
         assert np.add in qh.UFUNC_HELPERS
         assert np.add not in qh.UNSUPPORTED_UFUNCS
 
+    def test_thread_safety(self, fast_thread_switching):
+        def dummy_ufunc(*args, **kwargs):
```

Comment:
Would it make sense to just create a new helper instance here, so that we are sure the test does not change the main `UFUNC_HELPERS`?

Reply:
That is what I do, a few lines further down (`helpers = UfuncHelpers()`).

Reply:
Duh...